### ISSUE TITLE
TXT_KEY_EUI_BUILT_IN_X_CITIES in vanilla

### ISSUE THREAD

**Issue by TheRealAyCe** [LoneGazebo/Community-Patch-DLL#5105]
_1. Mod version (i.e Date - 4/23):_
Vox Populi (1-15-2).exe

_2. Mod list (if using Vox Populi only, leave blank):_

_3. Error description:_ 
There is an error text shown in vanilla (G&K+BNW) when getting a building tooltip.

_4. Steps to reproduce (optional):_

**Comment by Bloodrocution** [comment]
I have this issue too, but in different lines of text than your case.
![walls](http://puu.sh/CBEHV/9f0f221a49.jpg)
![barracks](http://puu.sh/CBEKL/94bc6201ac.jpg)

**Comment by TheRealAyCe** [comment]
6a/6b is needed for vanilla? I don't have a different version installed (VP and stuff is the only mod I ever installed so far).

**Comment by LoneGazebo** [comment]
If you leave EUi installed while playing vanilla, you need to follow EUi install instructions. Otherwise you need to remove it.

**Comment by TheRealAyCe** [comment]
I was under the impression that EUI is just a mod like the others. Apparently it is not. Where can I find "EUi install instructions"?

**Comment by LoneGazebo** [comment]
https://forums.civfanatics.com/resources/civ5-enhanced-user-interface.24303/

**Comment by TheRealAyCe** [comment]
I'm still not sure what I need to do. Am I right assuming the the VP installer does not install EUI correctly for using it with the base game? Because I have done no manual installing and any errors in the way it is installed would be a result of the VP installer. I did install 1-15-2 over 1-15 though.

**Comment by Bloodrocution** [comment]
When you say "base game" do you mean unmodded Civ without VP installed? In that case then yes, you'd need to use the manual installation of the EUI. However, if you want to use the EUI with VP, then you need to use the EUI that comes packaged with the VP installer and make sure that 6a is enabled. 

I was able to solve my UI issue by deleting the UI_bc1 folder (located at C:\Program Files (x86)\Steam\SteamApps\common\Sid Meier's Civilization V\Assets\DLC ) and then re-running the installer.

**Comment by TheRealAyCe** [comment]
I meant "base game" as in "unmodded Civ with VP installed but not used for this game". For playing VP I had no problems: Just tick all the mods and I'm good to go. But when I went back and wanted to play a vanilla game I noticed these text errors. I thought that, VP being a mod and all, I could just choose if I want the features or not, because VP+EUI has the additional 6a/6b mods to make it work. So I figured that without 6a/6b EUI would just work as intended in the base game, which is why I reported the error here. :)

**Comment by Bloodrocution** [comment]
Gotcha. Because the UI is integrated into the game's core installation files, VP's EUI (the **UI_bc1** folder) needs to be removed and replaced with the manual installation version if you want to go back to playing vanilla. You just have to alternate between installation methods each time you swap between VP and vanilla.

**Comment by LoneGazebo** [comment]
Yeah, EUI is different. I didn’t make the mod, but we distribute it with VP.

**Comment by TheRealAyCe** [comment]
Thank you. I checked and the "base EUI" is vastly different from the one used in VP. Too bad...
